TBouder/bowswap_ui
# Feature/bowswapv2

## What it does ✨
- Add the support for the new type of pools to match the latest version of the SmartContract with `metapoolSwapTokens` and `swapTokens`.
- Fix some flow issue
- Improve the fetching of price from coingecko with swr
- Fix issue when it was impossible to remove a char from the search input in the vault selection modal
- Adding the paths to handle the swap from vaults to vaults (other than metapoolSwap)

## How to test ✅
- Start your mainnetFork, setup the env variables with the contract deployed on mainnet
- Try to perform some swap, they should be successful !

## Note 🌾
We need to find a way to nicely update the paths and the supported FROM - TO pairs
